src/Modules/Verto/index.ts

```diff
@@ -8,6 +8,7 @@ import { trigger } from './services/Handler';
 import { localStorage } from './util/storage';
 import VertoHandler from './webrtc/VertoHandler';
 import { isValidOptions } from './util/helpers';
+import webrtcAdapter from 'webrtc-adapter';
```

Comment:
is this unused?

Reply:
It works like a polyfill https://www.npmjs.com/package/webrtc-adapter

Reply:
It may be more obvious to just do `import 'webrtc-adapter';` in that case, or to log `webrtcAdapter .browserDetails.version` if a DEBUG flag is set